### PR TITLE
Added Cypress

### PR DESCRIPTION
**Pull request by remyvdwereld**
Added basic setup for Cypress with Github secret integration for user password

### CODE REVIEW

File: cypress/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Zaken frontend - Cypress integration
```
Comment (remcohoff) [inline cypress/README.md:1]: Would be nice to mention the use of Cypress in the 'global' Readme and add a link to this Cypress-readme

**Review by remcohoff — COMMENTED** [review]
When I run Cypress locally I get an video-file, shouldn't we add that file or folder to the gitignore?

**Comment by remyvdwereld** [comment]
> When I run Cypress locally I get a video-file, shouldn't we add that file or folder to the gitignore?

Definitely! Maybe it's even better to switch that off as default?

File: cypress/README.md (hunk 1 above)
Comment (remyvdwereld) [reply cypress/README.md:1]: Good one! Do you agree with this separate readme file just for Cypress or should I include it in the global one?

Comment (remcohoff) [reply cypress/README.md:1]: For readability I think it's good to have it separate, but with a clear link to the file from the global Readme.

**Review by remcohoff — APPROVED** [review]

**Pull request merged**